xml/operations-maintenance-controller-full_recovery_test.xml

```diff
@@ -46,11 +46,11 @@
    <title>Description of the testing environment</title>
    <para>The testing environment is very similar to the Entry Scale model.</para>
    <para>It used 5 servers: 3 Controllers and 2 computes.</para>
-   <para>The controller node have three disks. The first one is reserved for the system, while others are used for swift.</para>
+   <para>The controller node have three disks. The first one is reserved for the system, while others are used for &o_objstore;.</para>
    <note>
-     <para>During this Disaster Recovery exercise, we have saved the data on disk 2 and 3 of the swift controllers.</para>
-     <para>This allow to restore the swift objects after the recovery.</para>
-     <para>If these disks were to be wiped as well, swift data would be lost but the procedure would not change.</para>
+     <para>During this Disaster Recovery exercise, we have saved the data on disk 2 and 3 of the &o_objstore; controllers.</para>
+     <para>This allow to restore the &o_objstore; objects after the recovery.</para>
+     <para>If these disks were to be wiped as well, &o_objstore; data would be lost but the procedure would not change.</para>
```

Comment:
Can all this text above be wrapped so it fits in the window? :) not a blocker, obvs. But just helpful for later!